edgelet/contrib/config/linux/config.yaml

```diff
@@ -96,23 +100,30 @@ provisioning:
 # production environments.
 #
 # Settings:
-#     device_ca_cert   - path to the device ca certificate and its chain
-#     device_ca_pk     - path to the device ca private key file
-#     trusted_ca_certs - path to a file containing all the trusted CA
+#     device_ca_cert   - URI of the device ca certificate and its chain.
+#                        Optionally can be specified as a file path.
+#     device_ca_pk     - URI of the device ca private key file.
+#                        Optionally can be specified as a file path.
+#     trusted_ca_certs - URI containing all the trusted CA
 #                        certificates required for Edge module communication
+#                        Optionally can be specified as a file path.
+#
 # Note:
-#     The values for these fields could be specified as a file path or a URI. 
-#     URI is preferred as it provides flexibility in where and 
-#     how these resources are to be accessed.
+#     The values for these fields could be specified as a file path or a URI.
```

Comment:
I don't think there's any point to the "However, ..." sentence.

- We're going to support file paths forever anyway. The user who has file paths today is going to continue to use file paths.

- A new user can see we support both, but may still decide to use paths, and there's nothing wrong with this.

- If we introduce a new URI-only non-file scheme tomorrow, then the users who use that will have no choice but to use URIs. This has no bearing on the existing users that use files via paths.

So there's no reason to try to convince users to switch from file paths to URIs. It doesn't affect "product evolution" either way and "product evolution" isn't something users should care about either.

I think the whole note can be replaced with:

>The values of these fields can be specified as a "file://" URI like "file:///path/cert_key.pem" or a file path like "/path/cert_key.pem"

without any opinions on which is better.

Reply:
Done.